packages/tangle/messagefactory.go

```diff
@@ -344,8 +346,12 @@ func PrepareReferences(strongParents MessageIDs, issuingTime time.Time, tangle *
 		opinionCanBeExpressed := true
 		for strongParentBranchID := range strongParentBranchIDs {
 			referenceParentType, referenceMessageID, err := referenceFromStrongParent(tangle, strongParentBranchID, issuingTime)
+			// Explicitly ignore error since we can't create a like/dislike reference to the message.
+			// We need to simply ignore it and remove it from the strong parents.
 			if err != nil {
-				return nil, errors.Errorf("failed to determine valid reference from Branch with %s: %w", strongParentBranchID, err)
+				referenceNotPossible.Add(strongParent)
+				opinionCanBeExpressed = false
```

Comment:
This should be logged somehow.

Reply:
Triggering an error where the function is called